lib/agents/jawsm.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const runtimePath = require('../runtime-path');
+const ConsoleAgent = require('../ConsoleAgent');
+
+class JawsmAgent extends ConsoleAgent {
+  constructor(options) {
+    super(options);
+
+    // disable node error coloring stuff
+    this.cpOptions = {
+      env: {
+        NO_COLOR: 1
+      }
+    };
+
+  }
+
+  async evalScript(code, options = {}) {
+    return super.evalScript(code, options);
+  }
+
+  parseError(str) {
+    return super.parseError(str);
```

Comment:
Same here, also does the default error parsing fully work for you?

Reply:
At this point I'm not really emitting proper errors most of the time (other than a SyntaxError maybe), so yeah, it "works". I might need to change it in the future once some proper error handling is there